src/foraging_gui/Foraging.py

```diff
@@ -2324,6 +2324,7 @@ def _UpdateSuggestedWater(self,ManualWater=0):
             else:
                 ManualWaterVolume=0
             Earnedwater=BS_TotalReward+ManualWaterVolume
+            self.Earnedwater=Earnedwater
```

Comment:
Should we call this "Earned water" or "Water in Session"? I think "Water in Session" better reflects that its earned water and manually given water. This is the language the RAs use.

Reply:
Changed